projects/experimental/components/pdf-viewer/pdf-viewer.template.html

```diff
@@ -0,0 +1,54 @@
+<ng-container [ngTemplateOutlet]="isMobile ? mobile : desktop" />
+
+<ng-template #mobile>
+    <tui-app-bar>
+        <button
+            tuiButton
+            tuiSlot="left"
+            type="button"
+            (click)="context.$implicit.complete()"
+        >
+            {{ close() }}
+        </button>
+        <ng-container [ngTemplateOutlet]="content" />
+        <ng-container tuiSlot="right">
+            <ng-container [ngTemplateOutlet]="actions" />
+        </ng-container>
+    </tui-app-bar>
+</ng-template>
+
+<ng-template #desktop>
+    <header
+        tuiTheme="dark"
+        class="t-header"
+    >
+        <span class="t-title">
+            <ng-container [ngTemplateOutlet]="content" />
+        </span>
+        <div class="t-actions">
+            <ng-container [ngTemplateOutlet]="actions" />
+            <button
+                *ngIf="$any(context).closable || context.closeable"
+                appearance="glass"
+                tuiButtonClose
```

Comment:
Let's ditch this component in v5. Now that I've managed to persuade designers to drop custom size for it it is pretty much useless.